sources/packages/web/src/constants/routes/RouteConstants.ts

```diff
@@ -66,6 +66,13 @@ export const AESTRoutesConst = {
   SEARCH_STUDENTS: Symbol(),
   STUDENT_DETAILS: Symbol(),
   SEARCH_INSTITUTIONS: Symbol(),
+  PROFILE: Symbol(),
```

Comment:
Is it not too generic to call it just profile since it is for the entire AEST portal?
Maybe rename it to INSTITUTION_PROFILE? Same idea for the others.

Reply:
i agree to this point